djangopackages/djangopackages
# Handle archived projects

**How to test:**
- Run the package updater command, and check on admin if archived GitHub Repos are flagged as archived.
- Access the package detail page and check if you can see a warning informing if that package is archived.

**Description of my solution:**
Fixes #761 
- Added `repo_archived` field to Package model, which will be populated when calling the fetch_metadata method from GithubHandler.(There's no direct field that informs if a GH repository is archived on `github3.py < 1.0 `(however, there is one in the latest version)
- Added a warning bootstrap alert on the package detail page to inform if a package was archived.

**Screenshots:**
![image](https://user-images.githubusercontent.com/1657123/139503721-de43d591-ded2-49d8-bb99-27094e5d24e4.png)

![image](https://user-images.githubusercontent.com/1657123/139503787-c3c66ec2-3d34-4d2b-9fae-a9e2a8098c90.png)
